src/main/java/io/airlift/tpch/PartColumn.java

```diff
@@ -79,7 +79,7 @@ public String getString(Part part)
             },
 
     @SuppressWarnings("SpellCheckingInspection")
-    RETAIL_PRICE("p_retailprice", DOUBLE)
+    RETAIL_PRICE("p_retailprice", DECIMAL)
             {
                 public double getDouble(Part part)
```

Comment:
is this method needed? I would expect `getLong` here.

Reply:
I left the method here for completeness sake. Do you want to remove ``getDouble`` from here?